src/main/java/src/main/java/EventModel.java

```diff
@@ -0,0 +1,36 @@
+package src.main.java;
```

Comment:
```suggestion
package fr.esir.jxc.models;
```
➜ fais la modif avec Intellij, il devrait renommer les dossier à ta place. `src/main/java` c'est très bizarre pour le moment, ça te fait une architecture de dossier `src/main/java/src/main/java`